src/OmniSharp.DotNetTest/VSTestManager.cs

```diff
@@ -101,18 +106,18 @@ private static void VerifyTestFramework(string testFrameworkName)
             }
         }
 
-        public override GetTestStartInfoResponse GetTestStartInfo(string methodName, string testFrameworkName, string targetFrameworkVersion)
+        public override GetTestStartInfoResponse GetTestStartInfo(string methodName, string runSettings, string testFrameworkName, string targetFrameworkVersion)
```

Comment:
normally we do not break  the public APIs but this particular package is no published on nuget, so it's OK

Reply:
Even though these functions are public, TestManager and VSTestManager are actually internal classes, so there probably isn't any worry.

Reply:
I was more concerned about the changes to the over-the-wire interface. At least the stdio version of the server seems to treat arguments as optional, in which case I think everything remains compatible.